sql/hive/compatibility/src/test/scala/org/apache/spark/sql/hive/execution/HiveCompatibilitySuite.scala

```diff
@@ -790,14 +821,13 @@ class HiveCompatibilitySuite extends HiveQueryFileTest with BeforeAndAfter {
     "nullscript",
     "optional_outer",
     "orc_dictionary_threshold",
-    "orc_empty_files",
     "order",
     "order2",
     "outer_join_ppr",
     "parallel",
     "parenthesis_star_by",
     "part_inherit_tbl_props",
-    "part_inherit_tbl_props_empty",
+    //"part_inherit_tbl_props_empty", // TODO: results don't match
```

Comment:
This one and others that are commented out below are caused by differences of describe table. There are mainly three differences. First, for managed tables, we will EXTERNAL set to false in table properties. For tables that are not bucketed, we will have numBuckets set to 0 instead of -1. For tables that does not specify the file format, the output format will be org.apache.hadoop.hive.ql.io.IgnoreKeyTextOutputFormat instead of org.apache.hadoop.hive.ql.io.HiveIgnoreKeyTextOutputFormat (For Hive, it somehow replaced even if the explain output still shows IgnoreKeyTextOutputFormat). We do not need to block this PR for any of them.